### ISSUE TITLE
Keyboard not responding after flashing with 1.0.4

### ISSUE THREAD

**Issue by razvan-ciochina** [Massdrop/mdloader#27]
Hello,

I just flashed my keyboard with the applet-flash-samd51j18a.bin at https://github.com/Massdrop/mdloader/releases/tag/1.0.4 on windows 10. After doing so my keyboard is completely dark even though the flashing went fine. 
Is there any way to recover it or is it bricked? 
My command prompt is here: https://ibb.co/2dj1R30

Thank you.

**Comment by razvan-ciochina** [comment]
Hi again,
Went through the code and finally understood what's what after also looking up what an applet is. The applet-flash-samd51j18a.bin is actually a binary that is selected by the mdloader application based on what keyboard you have (i.e. what type of MCU is on your keyboard). This applet is used to communicate with they keyboard, it's some data sent over the wire to your keyboard so that it gets in the right state so that it's ready to be flashed.
When I did the incredibly stupid thing of trying to flash the applet as the firmware for the kebyoard of course it was confused, that binary information was not what the keyboard was expecting.
The FILE_NAME in the steps is actually the keyboard firmware + config which you obtain by accessing the website here: https://drop.com/mechanical-keyboards/configurator/preset/ctrl--default
So long story short the bootloader got started by pressing the small hole in the back of the keyboard and I was able to flash again.
I'm going to leave this here in case someone makes the same stupid mistake.
